app/views/request/_sidebar.html.erb

```diff
@@ -9,32 +9,32 @@
                locals: { info_request: @info_request } %>
   <% end %>
 
-  <div class="request__special-status">
```

Comment:
I would split this into two commits, one where the indentation is sorted out, the second where the empty div is corrected. Would give a better diffs.

Reply:
Will fix in merge 👍